script/testing/junit/run_junit.py

```diff
@@ -69,16 +69,21 @@ def _wait_for_db_server(self):
         """ Wait for the db_server server to come up.
         """
         s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        # max wait of 10s in 0.1s increments
-        for i in range(100):
+        # max wait of 15s in 0.1s increments
+        start_time = time.time()
+        for i in range(150):
             try:
                 s.connect((self.db_server_host, self.db_server_port))
                 s.close()
                 print ("connected to server in {} seconds".format(i*0.1))
+                print("--- %s seconds ---" % (time.time() - start_time))
                 return
             except:
                 time.sleep(0.1)
                 continue
+        # self._stop_db_server()
+        print("--- %s seconds ---" % (time.time() - start_time))
+        print('DBMS fails to start up in 15s')
         return
```

Comment:
Shouldn't this return an error to stop the script?

Reply:
Yeah now an exception is raised